Index: src/main/java/Communication/Communication.java
===================================================================
--- src/main/java/Communication/Communication.java	(revision 121ec0e84e19c52c7ae9e1c3371e64ada4a6c4a0)
+++ src/main/java/Communication/Communication.java	(revision 121ec0e84e19c52c7ae9e1c3371e64ada4a6c4a0)
@@ -1,12 +0,0 @@
-package Communication;
-
-public class Communication {
-    public void Send(String ip,int port,String message){
-
-    }
-    public String Receice(){
-        String result = "";
-
-        return result;
-    }
-}
