Index: src/main/java/Kerberos/TCP.java
===================================================================
--- src/main/java/Kerberos/TCP.java	(revision ff7fc11ac06b9609a937056a7cf89c23802d9c3d)
+++ src/main/java/Kerberos/TCP.java	(revision ff7fc11ac06b9609a937056a7cf89c23802d9c3d)
@@ -1,10 +0,0 @@
-package Kerberos;
-
-public class TCP {
-    /*
-    封装Sockt TCP的发送和接收
-    两个方法：
-    1 Send()
-    2 Receive()
-     */
-}
